session/src/main/java/org/apache/iotdb/session/pool/SessionPool.java

```diff
@@ -1380,6 +1380,43 @@ public void createSchemaTemplate(Template template)
     }
   }
 
+  /**
+   * Create a template with flat measurements, not tree structured. Need to specify datatype,
+   * encoding and compressor of each measurement, and alignment of these measurements at once.
+   *
+   * @oaram templateName name of template to create
+   * @param measurements flat measurements of the template, cannot contain character dot
+   * @param dataTypes datatype of each measurement in the template
+   * @param encodings encodings of each measurement in the template
+   * @param compressors compression type of each measurement in the template
+   * @param isAligned specify whether these flat measurements are aligned
+   */
+  public void createSchemaTemplate(
+      String templateName,
+      List<String> measurements,
+      List<TSDataType> dataTypes,
+      List<TSEncoding> encodings,
+      List<CompressionType> compressors,
+      boolean isAligned)
```

Comment:
Is there "isAligned" parameter in the interface of v0.12? If not, should we add a same interface as that of 0.12 and use default isAligned = false ?

Reply:
Thanks! Now there is a completely compatible interface to creaete schema template with 0.12 in session